### ISSUE TITLE
Time delag/lag when adding new tasks to work items

### ISSUE THREAD

**Issue by Svirwani-Inteli** [figueiredorui/1-click-child-links#138]
I have noticed that there is at least a 15-20 seconds delay till the tasks get generated after clicking the extension. Is anyone else facing this delay?

**Comment by AndrewRedstone** [comment]
Yes, I am getting the same. started happening a week or 2 ago, you press the one click and seems like 20 seconds before it starts adding the tasks and then its taking about 2 seconds a task,

**Comment by JamieSandell** [comment]
Also have this issue.

**Comment by priyarani477** [comment]
Yes, I am also facing the same problem

**Comment by errmm** [comment]
Yeah I'm getting the same.  It's on the verge of being unusable.  I use this during calls and everyone is sitting there waiting as we wait 20 seconds for it to go ahead and start creating the child tasks...  Anyone got a 'fix' or using other add-in?

**Comment by ssteiner1** [comment]
Also having this issue.

**Comment by CommanderAlchemy** [comment]
Has anyone found a solution to this? 
This extension is the only tool that makes Devops actually usable without 500+ clicks...
